docs/spec/WIP_slashing/transactions.md

```diff
@@ -0,0 +1,19 @@
+
+### TxProveLive
+
+If a validator was automatically unbonded due to liveness issues and wishes to
+assert it is still online, it can send `TxProveLive`:
+
+```golang
+type TxProveLive struct {
+    PubKey crypto.PubKey
+}
+```
+
+All delegators in the temporary unbonding pool which have not
+transacted to move will be bonded back to the now-live validator and begin to
```

Comment:
"transacted to move" => "elected to unbond or redelegate"?

Reply:
Also, the validator may not necessarily be bonded again once unrevoked, they could have been replaced by another with higher power in the interim.

Reply:
WIP, another PR